src/condition/lfi_detector.cpp

```diff
@@ -99,9 +102,9 @@ lfi_result lfi_impl(std::string_view path, const ddwaf_object &params,
     const exclusion::object_set_ref &objects_excluded, const object_limits &limits,
     ddwaf::timer &deadline)
 {
-    if (path.find('/') == std::string_view::npos) {
+    if (path.find('/') == npos) {
```

Comment:
the comment in the next line seems to confuse `/` used in file names and `/` used in path separators